accounts/abi/utils.go

```diff
@@ -29,8 +32,12 @@ import "fmt"
 //
 // Name conflicts are mostly resolved by adding number suffix. e.g. if the abi contains
 // Methods "send" and "send1", ResolveNameConflict would return "send2" for input "send".
+// If a method name starts with a number an m is prepended (e.g. 1method -> m1method).
 func ResolveNameConflict(rawName string, used func(string) bool) string {
 	name := rawName
+	if unicode.IsDigit(rune(name[0])) {
+		name = fmt.Sprintf("%s%s", "m", name)
+	}
 	ok := used(name)
 	for idx := 0; ok; idx++ {
 		name = fmt.Sprintf("%s%d", rawName, idx)
```

Comment:
This would also need to use the `m`-prefixed version. So maybe you need to set `rawName` to name

Reply:
Damn now I see it, thanks will add a text